test/cpp/jit/test_interpreter.cpp

```diff
@@ -5,27 +5,71 @@ namespace torch {
 namespace jit {
 
 void testInterp() {
-  constexpr int batch_size = 4;
-  constexpr int input_size = 256;
-  constexpr int seq_len = 32;
+  {
```

Comment:
Nit: It would probably make sense to move the typecheck tests into its own function.